docs/annotate.rst

```diff
@@ -53,7 +57,7 @@ Step-by-step
 
```

Comment:
> Choose the Tag option in the right-hand ``General`` tab and click ``[ + ]`` to launch the Tag dialog

- `Choose the Tag option` should be `Choose the Tags option`
- `launch the Tag dialog` should perhaps be `launch the Tags Selection dialog` if you want to match the dialog title?

> Choose the existing Metaphase

`Choose the existing ``Metaphase`` tag`?

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/8/commits/3bf56ba5d06518443193797d8206c542bbc96a39